docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-dynamic-service-task-API-proc.adoc

```diff
@@ -0,0 +1,58 @@
+[id='case-management-dynamic-service-task-API-proc']
+= Creating a dynamic service task using the REST API
+
+Service tasks are generally less complex than user tasks, though they might need more data to be provided to execute properly. Service tasks require following information:
+
+* `name`: The name of the activity.
```

Comment:
I hate this, but IBM says to use this format for lists:
Data: the map... (no full stop unless at list one list item is a complete sentence

Reply:
This is kind of a definition list, for which IBM says "Capitalize the first word of each term or descriptive name, unless it usually would not be capitalized, for example, _num_circles_ or _prodname_ and capitalize the first word of each definition." 


The only solution I can think of here if it's really an issue is to format this like a terms list (ie, using :: to create a variable list), but I would still not be capitalizing these terms.